### ISSUE TITLE
The methods are called on object without acquisition that may lead to error

### ISSUE THREAD

**Issue by gbastien** [plone/plone.app.contentlisting#47]
Hi @jensens @mauritsvanrees 

Problem with https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/serializer/summary.py#L98 the plone.restapi summary serializer.

In `RealContentListingObject.__getattr__`, the attribute is returned with im_self without acquisition, that leads to having for example `<bound method MyObject.created of <MyObject at my-object-1>>` instead `<bound method MyObject.created of <MyObject at /Plone/folder/my-object-1>>`

In `RealContentListingObject.__getattr__` we should check if attr exists in aq_base(obj) but we should finally getattr on obj with acquisition:

```
if getattr(aq_base(obj), name, MARKER) is not MARKER:
    return getattr(obj, name)
```
Does that seems correct to you?  May I propose a PR for this?

Gauthier

**Comment by mauritsvanrees** [comment]
Seems good to me.

I would use `plone.base.utils.base_hasattr` and then `if base_hasattr(obj, name): return getattr(obj, name)`

**Comment by gbastien** [comment]
Hi @mauritsvanrees 

Thank you for feed back, I will provide a PR soon.

Gauthier

**Comment by mauritsvanrees** [comment]
Fixed by @gbastien in PR #48. Released just now in [3.0.1](https://pypi.org/project/plone.app.contentlisting/3.0.1/) which will be in Plone 6 rc2 today.

**Comment by mauritsvanrees** [comment]
Thanks!
